ts/parts-map/ColorAxis.ts

```diff
@@ -188,6 +210,9 @@ extend(ColorAxis.prototype, {
      * a true category. However, when your data is categorized, it may be as
      * convenient to add each category to a separate series.
      *
+     * Color axis does not work with: `sankey`, `sunburst`, `dependencywheel`,
+     * `networkgraph` and `gauge` series types.
+     *
```

Comment:
`object` and `*` become the same, if it is a parent option.

Reply:
I mean we're missing a description, something like this:
> In versions prior to v7.2.0, colorAxis could be configured **only** as a literal object.

Reply:
Okay. I would prefer to say that array is supported since v7.2.0. That has a more positive vibe imho. ;)
> Since v7.2.0, colorAxis can also be an array of options objects.

Reply:
Totally agree :)